Covid Patients/ViewController.swift

```diff
@@ -80,6 +80,16 @@ class ViewController: NSViewController, NSTableViewDelegate, NSTableViewDataSour
             }
         }
     }
+    @IBAction func detailsClicked(_ sender: NSButton) {
+        guard let row = lastRowSelected else{
```

Comment:
no need to have save selected row
should always be read from table 
this to avoid data sync issues

Reply:
should also validate the row is valid